core/modules/system/system.module

```diff
@@ -2202,6 +2202,7 @@ function system_init() {
   // Add the CSS for this module. These aren't in system.info, because they
   // need to be in the CSS_SYSTEM group rather than the CSS_DEFAULT group.
   backdrop_add_css('core/misc/normalize.css', array('group' => CSS_SYSTEM, 'every_page' => TRUE, 'weight' => -1000));
+  backdrop_add_css('core/misc/opinionate.css', array('group' => CSS_SYSTEM, 'every_page' => TRUE, 'weight' => -1001));
```

Comment:
Hmm, I thought that the weight number works like the rest of the Drupal/Backdrop weights do:

10
9
...
2
1
0
-1
-2
...
-9
-10